Mirror/Runtime/NetworkAnimator.cs

```diff
@@ -231,65 +231,63 @@ internal void HandleAnimTriggerMsg(int hash)
 
         void WriteParameters(NetworkWriter writer, bool autoSend)
         {
-            for (int i = 0; i < m_Animator.parameters.Length; i++)
+            AnimatorControllerParameter[] parameters = m_Animator.parameters;
```

Comment:
could you please add a comment above this line to indicate why we cache it and how big the difference is?

Reply:
Added :)